django-data/image/submissions/views.py

```diff
@@ -22,9 +23,9 @@
 from common.helpers import get_deleted_objects
 from common.views import OwnerMixin
 from cryoweb.tasks import import_from_cryoweb
-from image_app.models import Submission, Name
+from image_app.models import Submission, Name, ValidationSummary
```

Comment:
`ValidationSummary` imported but not used?

Reply:
Fixed